### ISSUE TITLE
XML-decryption module not found

### ISSUE THREAD

**Issue by ChessSpider** [RocketChat/Rocket.Chat#13271]
### Description:

Breaking bug: xml-encryption module not found on docker upon login

### Steps to reproduce:

1. Login via saml
2. fatal error:
```
Sorry, an annoying error occured
Error: Cannot find module 'xml-encryption'
Close Window
```
on docker
### Expected behavior:

No crash

### Actual behavior:

```
Sorry, an annoying error occured
Error: Cannot find module 'xml-encryption'
Close Window
```

### Server Setup Information:
```
rocketchat_1          | ➔ +-------------------------------------------------+
rocketchat_1          | ➔ |                  SERVER RUNNING                 |
rocketchat_1          | ➔ +-------------------------------------------------+
rocketchat_1          | ➔ |                                                 |
rocketchat_1          | ➔ |  Rocket.Chat Version: 0.74.0                    |
rocketchat_1          | ➔ |       NodeJS Version: 8.11.4 - x64              |
rocketchat_1          | ➔ |             Platform: linux                     |
rocketchat_1          | ➔ |         Process Port: 3000                      |
rocketchat_1          | ➔ |             Site URL: https://c***********  |
rocketchat_1          | ➔ |     ReplicaSet OpLog: Enabled                   |
rocketchat_1          | ➔ |          Commit Hash: cc5391e58c                |
rocketchat_1          | ➔ |        Commit Branch: HEAD                      |
rocketchat_1          | ➔ |                                                 |
rocketchat_1          | ➔ +-------------------------------------------------+
```

- Version of Rocket.Chat Server: 
- Operating System:  linux
- Deployment Method: docker
- Number of Running Instances: 1
- DB Replicaset Oplog: 
- NodeJS Version: 
- MongoDB Version: 

### Additional context

Login via simplesaml

**Comment by ChessSpider** [comment]
Probably related to https://github.com/RocketChat/Rocket.Chat/pull/12153

**Comment by zenman94** [comment]
Same here in docker release.

**Comment by ChessSpider** [comment]
@zenman94 temporary fix:
copy/paste: 
```
docker-compose exec rocketchat /bin/bash
cd /app/bundle/programs/server
npm install xml-encryption
exit
```

**Comment by zenman94** [comment]
Thank you for the patch but we are on Kubernetes cluster.

**Comment by hairmare** [comment]
Looks like xml-encryption was removed in 687c3c572aaba7b7cb67ea5683b3100f54bfcb53 while it is still used here:
https://github.com/RocketChat/Rocket.Chat/blob/483f7d5c752679a9795f6324a73e7c03f36601fc/packages/meteor-accounts-saml/server/saml_utils.js#L396
If you are using the [`stable/rocketchat` helm chart](https://github.com/helm/charts/blob/master/stable/rocketchat/) to deploy rocketchat you can set `securityContext.runAsUser=99999` and then follow the example from `helm status <rocketchat-release>` to open a shell where the workaround from https://github.com/RocketChat/Rocket.Chat/issues/13271#issuecomment-458034489 works.

**Comment by zenman94** [comment]
We prefer wait a correct docker image before push this in production :)

**Comment by ChessSpider** [comment]
when will this be fixed?

**Comment by GuilleGlad** [comment]
What solution do you recommend if I'm using a Snap but I need a temporary solution until the stable span is released? thank you.

**Comment by adanielvv** [comment]
Temporary workaround for snap:
`sudo snap revert rocketchat-server`

You should always make a backup first:
```
sudo systemctl stop snap.rocketchat-server.rocketchat-server
sudo rocketchat-server.backupdb
sudo snap revert rocketchat-server
sudo systemctl restart snap.rocketchat-server.rocketchat-server
```

**Comment by GuilleGlad** [comment]
Thank you so much @adanielvv , is there an option to disable automatic SNAP updates?

**Comment by adanielvv** [comment]
@GuilleGlad I looked at the [snap manpages](https://manpages.ubuntu.com/manpages/artful/man1/snap.1.html) and couldn't find a hold option.
I found a command option that shows the impending updates:
`sudo snap refresh --list`
On my system (after the revert) it comes up empty.
I guess it jumps the reverted version.

By the way, the snap revert command is listed in the [Rocket.Chat docs](https://rocket.chat/docs/installation/manual-installation/ubuntu/snaps/#how-do-i-revert-to-the-previous-version-of-rocketchat). See also the manpage above to understand more fully what happens.
